core/src/main/java/org/dst/core/operatorImpl/DstSetImpl.java

```diff
@@ -33,6 +33,16 @@ public boolean del(String key) {
     return true;
   }
 
+  @Override
+  public boolean del(String key, String value) {
```

Comment:
It's better to return `KEY_NOT_FOUND` status.

Reply:
Fine.

Reply:
Done.